### PR TITLE
install_action_async: Use owned Variant in closure

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

File: gtk4/src/subclass/widget.rs [hunk 1]
```diff
@@ -926,7 +926,7 @@ pub unsafe trait WidgetClassSubclassExt: ClassStruct {
                 let action_name = action_name.to_owned();
                 let parameter_type = parameter_type.map(ToOwned::to_owned);
                 ctx.spawn_local(glib::clone!(@strong this, @strong action_name, @strong parameter_type, @strong activate => async move {
-                    activate(this, &action_name, parameter_type.as_ref()).await;
+                    activate(this, &action_name, parameter_type).await;
```
Comment (sdroege) [inline gtk4/src/subclass/widget.rs:929]: Might also want to pass `action_name` in there directly instead of borrowing?

**Comment by sdroege** [comment]
CI is fixed by https://github.com/gtk-rs/gtk4-rs/pull/1227

File: gtk4/src/subclass/widget.rs (hunk 1 above)
Comment (A6GibKm) [reply gtk4/src/subclass/widget.rs]: Done

**Comment by bilelmoussaoui** [comment]
Please reformat the code

**Comment by A6GibKm** [comment]
> Please reformat the code

Oops, done.

**Pull request merged**